tensorboard/webapp/app_routing/effects/app_routing_effects_test.ts

```diff
@@ -655,16 +665,17 @@ describe('app_routing_effects', () => {
               pathname: '/experiments',
             }),
             // From getActiveNamespaceId().
-            beforeNamespaceId: before.toString(),
+            beforeNamespaceId: `1234:${beforeDate}`,
             // Value of before + the 5000 milliseconds from tick(5000).
-            afterNamespaceId: (before + 5000).toString(),
+            afterNamespaceId: `023c:${beforeDate + 5000}`,
           }),
         ]);
       }));
 
       it('are reused on navigationRequested when resetNamespacedState is false', fakeAsync(() => {
         // Record initial time for use later.
-        const before = Date.now();
+        // '023c' is the string with base 16 convertd from TEST_UNIT8ARRAY
+        const before = `023c:${Date.now()}`;
```

Comment:
I prefer the type of change you made in other places, where this was renamed beforeDate and then the random id and beforeDate were combined later in the expect() call.